tasks/main.yml

```diff
@@ -1 +1,21 @@
-# role tasks
\ No newline at end of file
+# role tasks
+
+- name: open ports for zone "{{ zone }}" (IPv4+IPv6)
```

Comment:
Is `"{{ zone }}"` the proper thing to have here?

Reply:
It's not.  Will fix.